### PR TITLE
Extract docs from README.md and create pipeline for publishing into Wiki

### PR DESCRIPTION
**Pull request by Zwaen91**
The Readme file is becoming too long, so we should build a better structure.
As the wiki is maintained independently of code changes, we should place them "in code" in this case the docs folder. 
For fans of a wiki page, the contents of the docs folder will be published in the wiki.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,159 +15,5 @@ tightly tailored to the needs of our team members working on various projects wi
 - https://devenv.sh/
 - https://developer.shopware.com/docs/guides/installation/devenv
 
-## Setup
```
Comment (jinnoflife) [inline README.md:18]: Can we add a "look at the wiki" hint here 🙂 .

Comment (Zwaen91) [reply README.md:18]: Good point. What's your opinion about the other menu entries moved from the readme to docs?

Comment (jinnoflife) [reply README.md:18]: Sorry, I meant all points - for the sake of simplicity I had only picked out the top one 🙂 

I think that through your change, the wiki is now present enough 👍

**Review by jinnoflife — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Zwaen91) [reply README.md:18]: I moved "Setup & Usage" one up to make in even more present

**Review by jochenmanz — APPROVED** [review]

**Pull request merged**